.circleci/config.yml

```diff
@@ -4184,7 +4184,7 @@ jobs:
 
     - run:
         name: Run Operator Bundle Scorecard tests
-        command: make -C operator bundle-test-image
+        command: ./operator/scripts/retry.sh 4 make -C operator bundle-test-image
```

Comment:
How about using the retry function in `./operator/hack/common.sh` instead?

Reply:
Changed so that `operator/scripts/retry.sh` is merely a wrapper exposing the library function in `common.sh` to the command line.